utils.js

```diff
@@ -0,0 +1,11 @@
+import * as fs from 'fs'
+export function getAuthData(){
+    const filePath = "creds.json";
+    const rawData = fs.readFileSync(filePath);
+    const jsonData = JSON.parse(rawData.toString());
+
+    return {
```

Comment:
Its easier in this way:
```suggestion
    return jsonData;
```